{
  "configurations": {
    "CVE_data_version": "4.0",
    "nodes": []
  },
  "cve": {
    "CVE_data_meta": {
      "ASSIGNER": "cve@mitre.org",
      "ID": "CVE-2022-40896"
    },
    "data_format": "MITRE",
    "data_type": "CVE",
    "data_version": "4.0",
    "description": {
      "description_data": [
        {
          "lang": "en",
          "value": "A ReDoS issue was discovered in pygments/lexers/smithy.py in pygments through 2.15.0 via SmithyLexer."
        }
      ]
    },
    "problemtype": {
      "problemtype_data": [
        {
          "description": []
        }
      ]
    },
    "references": {
      "reference_data": [
        {
          "name": "https://pypi.org/project/Pygments/",
          "refsource": "MISC",
          "tags": [],
          "url": "https://pypi.org/project/Pygments/"
        },
        {
          "name": "https://pyup.io/posts/pyup-discovers-redos-vulnerabilities-in-top-python-packages-part-2/",
          "refsource": "MISC",
          "tags": [],
          "url": "https://pyup.io/posts/pyup-discovers-redos-vulnerabilities-in-top-python-packages-part-2/"
        },
        {
          "name": "https://github.com/pygments/pygments/blob/master/pygments/lexers/smithy.py#L61",
          "refsource": "MISC",
          "tags": [],
          "url": "https://github.com/pygments/pygments/blob/master/pygments/lexers/smithy.py#L61"
        }
      ]
    }
  },
  "impact": {},
  "lastModifiedDate": "2023-07-19T16:55Z",
  "publishedDate": "2023-07-19T15:15Z"
}